src/test/java/core/basesyntax/FileWriterImplTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+
+public class FileWriterImplTest {
+
+    private final FileWriterImpl fileWriter = new FileWriterImpl();
+
+    @Test
+    public void writeFile_validPath_ok() throws Exception {
+        String filePath = "src/test/resources/output-file.csv";
+        String data = "fruit,quantity\napple,10\nbanana,20";
+        Files.createDirectories(Paths.get("src/test/resources"));
+        fileWriter.write(data, filePath);
+        assertTrue(Files.exists(Paths.get(filePath)));

```

Comment:
It would be good to check the content of the file. But don't use the method of FileReaderImpl for this. Create a private method in this class. 
Also, let's create test with invalid path